### ISSUE TITLE
Buoyancy test is not running

### ISSUE THREAD

**Issue by jginternational** [KratosMultiphysics/GiDInterface#775]
Buoyancy test Heated Square 2D is not calculating.
The calc ends with no results

**Comment by jginternational** [comment]
Changing the fluid element to  Dynamic VMS everything runs.
So problem seem to appear with Quasi static VMS

**Comment by rubenzorrilla** [comment]
There is a memory leak because there are no `constitutive_law` defined in the `FluidMaterials.json`. This doesn't happen in the standard fluid interface.

Concerning your observation about the dynamic VMS, the point is that switching to the Dynamic VMS doesn't write the `formulation` field. Same happens with the FIC element. Hence it's taking the default in the Kratos solver, which doesn't require a constitutive law.

Long story short, we have two errors:
1) The `FluidMaterials.json` doesn't contain the `constitutive_law` entry.
2) The `formulation` dictionary in the `solver_settings` of the fluid is not written when selecting the dynamic VMS or the FIC elements. This seems to not happen in the standard fluid GUI. I guess it is related to #767
